web-registry/src/App.tsx

```diff
@@ -102,8 +103,12 @@ const App: React.FC = (): JSX.Element => {
             element={<PostPurchase />}
           />
           <Route
-            path="/dashboard/ecocredits"
-            element={<ProtectedRoute component={Ecocredits} />}
+            path="ecocredits/dashboard"
+            element={<ProtectedRoute component={MyEcocredits} />}
```

Comment:
this one shouldn't be protected either (we only need the keplr wallet to be connected for now, and not the user to be logged in with auth0)